imagebitmap-renderingcontext/context-creation-with-alpha.html

```diff
@@ -0,0 +1,92 @@
+    function consumeImageBitmapAlphaDefault(image) {
+        var dstCanvas = document.createElement('canvas');
+        dstCanvas.width = width;
+        dstCanvas.height = height;
+        var dstCtx = dstCanvas.getContext('bitmaprenderer');
+        dstCtx.transferFromImageBitmap(image);
+
+        var myCanvas = document.createElement('canvas');
+        myCanvas.width = width;
+        myCanvas.height = height;
+        var myCtx = myCanvas.getContext('2d');
+        myCtx.drawImage(dstCanvas, 0, 0);
+        testCanvas(myCtx, 0, 255, 0, 127);
+    }
+
+    var srcCanvas = document.createElement('canvas');
+    srcCanvas.width = width;
+    srcCanvas.height = height;
+    var ctx = srcCanvas.getContext('2d');
+    ctx.fillStyle = 'rgba(0, 255, 0, 0.5)';
+    ctx.fillRect(0, 0, width, height);
+    createImageBitmap(srcCanvas).then(consumeImageBitmapAlphaDefault);
+}, "Test that canvas.getContext('bitmaprenderer') preserves alpha");
+
+promise_test(function() {
+    function consumeImageBitmapAlphaFalse(image) {
+        var dstCanvas = document.createElement('canvas');
+        dstCanvas.width = width;
+        dstCanvas.height = height;
+        var dstCtx = dstCanvas.getContext('bitmaprenderer', { alpha: false });
+        dstCtx.transferFromImageBitmap(image);
+
+        var myCanvas = document.createElement('canvas');
+        myCanvas.width = width;
+        myCanvas.height = height;
+        var myCtx = myCanvas.getContext('2d');
+        myCtx.drawImage(dstCanvas, 0, 0);
+        testCanvas(myCtx, 0, 127, 0, 255);
+    }
+
+    var srcCanvas = document.createElement('canvas');
+    srcCanvas.width = width;
+    srcCanvas.height = height;
+    var ctx = srcCanvas.getContext('2d');
+    ctx.fillStyle = 'rgba(0, 255, 0, 0.5)';
+    ctx.fillRect(0, 0, width, height);
+    createImageBitmap(srcCanvas).then(consumeImageBitmapAlphaFalse);
+}, "Test that canvas.getContext('bitmaprenderer', alpha:false) sets the canvas opaque");
+
+promise_test(function() {
+    function consumeImageBitmapAlphaTrue(image) {
+        var dstCanvas = document.createElement('canvas');
+        dstCanvas.width = width;
+        dstCanvas.height = height;
+        var dstCtx = dstCanvas.getContext('bitmaprenderer', { alpha: true });
+        dstCtx.transferFromImageBitmap(image);
+
+        var myCanvas = document.createElement('canvas');
+        myCanvas.width = width;
+        myCanvas.height = height;
+        var myCtx = myCanvas.getContext('2d');
+        myCtx.drawImage(dstCanvas, 0, 0);
+        testCanvas(myCtx, 0, 255, 0, 127);
+    }
+
+    var srcCanvas = document.createElement('canvas');
+    srcCanvas.width = width;
+    srcCanvas.height = height;
+    var ctx = srcCanvas.getContext('2d');
+    ctx.fillStyle = 'rgba(0, 255, 0, 0.5)';
+    ctx.fillRect(0, 0, width, height);
+    createImageBitmap(srcCanvas).then(consumeImageBitmapAlphaTrue);
+}, "Test that canvas.getContext('bitmaprenderer', alpha:true) preserves the alpha");
```

Comment:
I think what you mean is "Test that an ImageBitmapRenderingContext with alpha enabled preserves alpha".